src/sammich/plugins/create_issue.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from dotenv import dotenv_values
+from github import Github
+from machine.plugins.base import MachineBasePlugin
+from machine.plugins.decorators import command
+
+import settings
+
+# Load secrets
+secrets = dotenv_values(".secrets")
+github_token = secrets["github_token"]
```

Comment:
Why do you need it here?

Reply:
where should I add it inside the function!!

Reply:
To a right place depending on the scope needed by your code. Check where you use it first.

Reply:
how about now !?